sirepo/pkcli/admin.py

```diff
@@ -77,9 +125,21 @@ def move_user_sims(target_uid=''):
         shutil.move(lib_file, target)
 
 
+def proprietary_sim_type_dir(sim_type):
+    return srdb.root().join(cfg.proprietary_code_dir, sim_type)
```

Comment:
This doesn't need to be a cfg. It's "our" namespace so we can do what we want with it, and it isn't something we can change by just by changing the cfg. 

proprietary_sim_type_dir is proprietary_code_dir. No need for a different name.